c_emulator/riscv_sim.c

```diff
@@ -709,14 +736,16 @@ void flush_logs(void)
 
 #ifdef RVFI_DII
 
-typedef void (packet_reader_fn)(lbits *rop, unit);
-static void get_and_send_rvfi_packet(packet_reader_fn *reader) {
+typedef void(packet_reader_fn)(lbits *rop, unit);
```

Comment:
Huh, that's some strange formatting for function types

Reply:
Fixed by adding a `*`